Arelle/Arelle
# Restore old linux build scripts

#### Reason for change
We need to retain the original Arelle build scripts until we've completely switched over to the CI builds.

#### Description of change
Restores the linux build scripts removed in the CI Linux builds PR

#### Steps to Test
* Generate CI linux builds and make sure the tar file still contains an Arelle build

**review**:
@Arelle/arelle
